content/src/ports/activeEntities.ts

```diff
@@ -80,15 +80,15 @@ export const createActiveEntitiesComponent = (
   }
 
   const setPreviousEntityAsNone = (pointer: string): void => {
-    if (entityIdByPointers.has(pointer)) {
```

Comment:
is this `toLowerCase()` expected? It seems like it wasn't there in 5.1.10 https://github.com/decentraland/catalyst/compare/5.1.10...5.1.12#diff-7de2cb7147f42a1e556a17d57070b61730d1c3262cae3154603ccd648477f39fL107

Reply:
miss-conflict resolution, I'm on it